hubblestack/modules/fdg.py

```diff
@@ -185,11 +194,13 @@ def fdg(fdg_file, starting_chained=None):
     ret = _fdg_execute('main', block_data, chained=starting_chained)
     return RETURNER_ID_BLOCK, ret
 
-def run(fdg_file=None, starting_chained=None):
+def run(fdg_file=None, starting_chained=None, fdg_version='fdg'):
```

Comment:
Since fdg.run was meant for v2 only. So default value here should be kept: 'fdg_v2'.
Else in every case to run new fdg, we need to pass extra param

Reply:
done